bff/pkg/server/server.go

```diff
@@ -120,11 +120,22 @@ func convertDbIdentifiersToPromQLParam(identifiers []string) string {
 }
 
 func getActualDbIdentifier(dbIdentifier string) (string, error) {
-	parts := strings.Split(dbIdentifier, "/")
-	if len(parts) < 2 {
-		return "", fmt.Errorf("invalid dbidentifier format: %s", dbIdentifier)
+	_, systemID, _, err := splitDbIdentifier(dbIdentifier)
+	return systemID, err
+}
+
+func splitDbIdentifier(dbIdentifier string) (string, string, string, error) {
+	parts := strings.SplitN(dbIdentifier, "/", 3)
+
+	if len(parts) != 3 {
```

Comment:
the string will actually have 4 parts 
amazon_rds / mohammad-dashti-rds-1 / us-west-2 / cvirkksghnig
you would then concat the last 2 parts into the systemScope

Reply:
The third pard (systemScope) might or might not have the `/`. It can be either `us-west-2` or `us-west-2/cvirkksghnig`. Please consider that in validation, too.